tox.ini

```diff
@@ -61,8 +60,8 @@ base_python = python3.12
 
 ; Note: the following are commented out for development convenience,
 ;       so as to test the interactive mode with a different Wagtail version
-; deps =
-;     wagtail>=5.2,<6.0
+deps =
+    wagtail>=5.2,<6.0
```

Comment:
**Question**: Is there a reason for uncommenting this?

Reply:
No, it's just me uncommenting it during testing and forgetting to comment again. Changed this in the newest commit.